code_samples/editor_basics/src/main/java/org/intellij/sdk/editor/EditorIllustration.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2000-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
+package org.intellij.sdk.editor;
+
+import com.intellij.openapi.actionSystem.*;
+import com.intellij.openapi.command.WriteCommandAction;
+import com.intellij.openapi.editor.*;
+import com.intellij.openapi.editor.actionSystem.*;
+import com.intellij.openapi.project.Project;
+
+/**
+ * Menu action to replace a selection of characters with a fixed string.
+ * @author Anna Bulenkova
+ * @see com.intellij.openapi.actionSystem.AnAction
+ */
+public class EditorIllustration extends AnAction {
```

Comment:
could we rename this class to `EditorIllustrationAction`

Reply:
Done.